Documentation/common-problems.md

```diff
@@ -4,22 +4,66 @@ weight: 78
 indent: true
 ---
 
-# Common Problems
+# Common Issues
```

Comment:
the file is called `common-problems.md`, should that be changed too?

Reply:
Yes, I'll change that too. The TOC already had a different name, which was bad enough